src/basic/namespace-util.c

```diff
@@ -199,8 +200,10 @@ int detach_mount_namespace(void) {
         return RET_NERRNO(mount(NULL, "/", NULL, MS_SLAVE | MS_REC, NULL));
 }
 
-int userns_acquire(const char *uid_map, const char *gid_map) {
+int userns_acquire(const char *uid_map, const char *gid_map, uid_t owner) {;
```

Comment:
What's the semicolon doing here?

Reply:
santa left it there on xmas eve because we have been such good boys